modules/lastseen.cpp

```diff
@@ -128,6 +130,7 @@ class CLastSeenMod : public CModule {
         return false;
     }
 
+    // Provides GUI to configure this module by adding a widget to user page in webadmin.
```

Comment:
I wasn't sure about this one so I will comment it out. Should I seperate one commit of updating all the dates and one of adding comments to modules?